pdr_backend/pdr_dashboard/dash_components/plots.py

```diff
@@ -3,23 +3,24 @@
 
 import plotly.graph_objects as go
 from enforce_typing import enforce_types
+from statsmodels.stats.proportion import proportion_confint
 
 from pdr_backend.cli.arg_feeds import ArgFeeds
 from pdr_backend.util.time_types import UnixTimeS
 
 
 @enforce_types
-def process_payouts(payouts: List[dict]) -> tuple:
+def process_payouts(payouts: List[dict], calculate_confint: bool = False) -> tuple:
     """
     Process payouts data for a given predictor and feed.
     Args:
         payouts (list): List of payouts data.
         predictor (str): Predictor address.
         feed (str): Feed contract address.
     Returns:
-        tuple: Tuple of slots, accuracies, profits, and stakes.
+        tuple: Tuple of slots, accuracies, profits, stakes.
     """
-    slots, accuracies, profits, stakes = [], [], [], []
+    slots, accuracies, profits, stakes, acc_intervals = [[] for _ in range(5)]
```

Comment:
at this point, I would recommend having a NamedTuple instead.
That way we can access the object by name. I remember doing this in ocean.py at some point, if you need inspo.